src/components/WebexActivity/WebexActivity.stories.js

```diff
@@ -0,0 +1,178 @@
+import React, {Fragment} from 'react';
+import {storiesOf} from '@storybook/react';
+import {addDays, getDay, subDays} from 'date-fns';
+
+import {ActivitiesJSONAdapter, PeopleJSONAdapter} from '../../adapters';
+import {activities, people} from '../../data';
+
+import WebexActivity from './WebexActivity';
+
+// Setup for the stories
+const [activityID] = Object.keys(activities);
+const stories = storiesOf('Webex Activity', module);
+const newActivities = {};
+
+// Stories
+stories.add('default', () => (
+  <WebexActivity
+    activityID={activityID}
+    adapters={{activitiesAdapter: new ActivitiesJSONAdapter(activities), peopleAdapter: new PeopleJSONAdapter(people)}}
+  />
+));
+
+stories.add('multi-line text', () => {
+  newActivities[activityID] = {...activities[activityID], text: 'line number1\nline number2\nline number3'};
+
+  return (
+    <WebexActivity
+      activityID={activityID}
+      adapters={{
+        activitiesAdapter: new ActivitiesJSONAdapter(newActivities),
+        peopleAdapter: new PeopleJSONAdapter(people),
+      }}
+    />
+  );
+});
+
+stories.add('long text', () => {
+  newActivities[activityID] = {
+    ...activities[activityID],
+    text:
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.',
+  };
+
+  return (
+    <WebexActivity
+      activityID={activityID}
+      adapters={{
+        activitiesAdapter: new ActivitiesJSONAdapter(newActivities),
+        peopleAdapter: new PeopleJSONAdapter(people),
+      }}
+    />
+  );
+});
+
+stories.add('multiple activities', () => (
```

Comment:
I think this is something that belong with the Activity Stream and not with activity